nginx/Dockerfile

```diff
@@ -25,8 +26,8 @@ ENV DOLLAR='$'
 # Substitute contents and write a new application.conf file, which is imported
 # by nginx.conf
 RUN envsubst \
-      </etc/nginx/conf.d/flask_app.conf \
-      >/etc/nginx/conf.d/application.conf \
+  </etc/nginx/conf.d/flask_app.conf \
+  >/etc/nginx/conf.d/application.conf \
```

Comment:
Can we revert back this change? I find it easier to read when indenting line breaks from within the same commands.
```suggestion
      </etc/nginx/conf.d/flask_app.conf \
      >/etc/nginx/conf.d/application.conf \
```